src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -1009,6 +1322,87 @@ instruct reduce_maxD(vRegD dst, vRegD src1, vReg src2) %{
 
 // vector min reduction
 
+instruct reduce_minB(iRegINoSp dst, iRegIorL2I src1, vReg src2, vRegD tmp, rFlagsReg cr) %{
+  predicate(UseSVE > 0 && n->in(2)->bottom_type()->is_vect()->length_in_bytes() >= 16 &&
+            (n->in(2)->bottom_type()->is_vect()->element_basic_type() == T_BYTE));
+  match(Set dst (MinReductionV src1 src2));
+  effect(TEMP_DEF dst, TEMP tmp, KILL cr);
+  ins_cost(SVE_COST);
+  format %{ "sve_sminv $tmp, $src2\t# vector (sve) (B)\n\t"
+            "smov  $dst, $tmp, B, 0\n\t"
+            "cmpw  $dst, $src1\n\t"
+            "cselw $dst, $dst, $src1 LT\t# min reduction B" %}
+  ins_encode %{
+    __ sve_sminv(as_FloatRegister($tmp$$reg), __ B,
+         ptrue, as_FloatRegister($src2$$reg));
+    __ smov($dst$$Register, as_FloatRegister($tmp$$reg), __ B, 0);
+    __ cmpw($dst$$Register, $src1$$Register);
+    __ cselw(as_Register($dst$$reg), as_Register($dst$$reg), as_Register($src1$$reg), Assembler::LT);
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+instruct reduce_minS(iRegINoSp dst, iRegIorL2I src1, vReg src2, vRegD tmp, rFlagsReg cr) %{
+  predicate(UseSVE > 0 && n->in(2)->bottom_type()->is_vect()->length_in_bytes() >= 16 &&
+            (n->in(2)->bottom_type()->is_vect()->element_basic_type() == T_SHORT ||
+            (n->in(2)->bottom_type()->is_vect()->element_basic_type() == T_CHAR)));
```

Comment:
As Andrew said in my patch review, does it make sense to have T_CHAR here: https://github.com/openjdk/jdk/pull/726#discussion_r507726726?

Reply:
I think it can be consistent with other structs first, and then modify together in the main line